awx/ui_next/src/components/Lookup/Lookup.test.jsx

```diff
@@ -1,285 +1,359 @@
 /* eslint-disable react/jsx-pascal-case */
 import React from 'react';
 import { createMemoryHistory } from 'history';
-import { mountWithContexts } from '../../../testUtils/enzymeHelpers';
+import { mountWithContexts, waitForElement } from '@testUtils/enzymeHelpers';
 import Lookup, { _Lookup } from './Lookup';
 
 let mockData = [{ name: 'foo', id: 1, isChecked: false }];
 const mockColumns = [{ name: 'Name', key: 'name', isSortable: true }];
-describe('<Lookup />', () => {
-  test('initially renders succesfully', () => {
-    mountWithContexts(
-      <Lookup
-        lookupHeader="Foo Bar"
-        name="fooBar"
-        value={mockData}
-        onLookupSave={() => {}}
-        getItems={() => {}}
-        columns={mockColumns}
-        sortedColumnKey="name"
-        multiple
-      />
-    );
-  });
 
-  test('API response is formatted properly', done => {
-    const wrapper = mountWithContexts(
-      <Lookup
-        lookupHeader="Foo Bar"
-        name="fooBar"
-        value={mockData}
-        onLookupSave={() => {}}
-        getItems={() => ({
-          data: { results: [{ name: 'test instance', id: 1 }] },
-        })}
-        columns={mockColumns}
-        sortedColumnKey="name"
-        multiple
-      />
-    ).find('Lookup');
+/**
+ * Check if an element exists on the document body
+ * @param {selector} query selector
+ * @param {expected} bool to assert existence or non-existence
+ */
+function checkRoot(selector, expected) {
```

Comment:
kind of an unclear method name. maybe split into two (instead of a bool param) and name them something like `assertElementFoundInDocument` and `assertElementNotFoundInDocument` or something similar?

Or, since you're only ever passing 'body div[role="dialog"]' as a selector, remove both params and make `assertModalVisible` and `assertModalNotVisible`

Reply:
I like that idea - I also like your unwavering contempt for boolean function params.

Reply:
fixed